backend/plugins/frontline_api/src/modules/integrations/imap/configs.ts

```
@@ -1,145 +1,182 @@
+                        if (toUpper(attachment.encoding) === 'BASE64') {
+                          stream.pipe(new Base64Decode()).pipe(writeStream);
+                        } else {
+                          stream.pipe(writeStream);
+                        }
+
+                        writeStream.on('finish', () => {
+                          res.download(
+                            tempPath,
+                            attachment.params.name,
+                            (err) => {
+                              fs.unlink(tempPath, () => {});
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)